io.github.daviddesimone.opencloudsaves.yml

```diff
@@ -0,0 +1,68 @@
+app-id: io.github.daviddesimone.opencloudsaves
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: bootstrap-opencloudsave.sh
+cleanup:
+  - /app/lib/sdk/golang
+finish-args:
```

Comment:
```suggestion
finish-args:
```

Reply:
I may be mistaken, but I think that this is no longer needed now that I am properly using the golang sdk extension.